lib/sisimai/bite/email/mailfoundry.rb

```diff
@@ -0,0 +1,133 @@
+module Sisimai::Bite::Email
+  module MailFoundry
+    # Sisimai::Bite::Email::MailFoundry parses a bounce email which created by
+    # MailFoundry. Methods in the module are called from only Sisimai::Message.
+    class << self
+      # Imported from p5-Sisimail/lib/Sisimai/Bite/Email/MailFoundry.pm
+      require 'sisimai/bite/email'
+
+      Re0 = {
```

Comment:
__[rubocop]__
* Style/MutableConstant: Freeze mutable objects assigned to constants.

<!-- Comment by [SideCI](https://sideci.com) -->